### ISSUE TITLE
Planning Mode: unit standing animation twitchy

### ISSUE THREAD

**Issue by matthiaskrgr** [wesnoth/wesnoth#3026]
for some reason, the unit displayed representing planned moves has a very twitchy idle animation (I assume some frames are skipped) while the display of the unit representing the last plan is stable/normal, seen on the human horseman or undead bat.

**Comment by gfgtdf** [comment]
i noticed that aswell.

**Comment by Wedge009** [comment]
Interesting, I just tried 1.14 (d4101b87) and debug-created some bats. Toggling between standard and planning modes doesn't make any difference for me.

**Comment by jostephd** [comment]
Tested with a Gryphon Rider. It looks like some frames are played in the unit's real/current location and some in the future location.

**Comment by Wedge009** [comment]
Maybe it's just because I'm running on a fairly fast CPU but I don't notice too much, in either normal or planning mode. But I do notice that when using the 'undo planned move' option with the Gryphon Rider that it leaves bits of its frames behind - parts of the sprite that go beyond the edges of the hex.

**Comment by jostephd** [comment]
> But I do notice that when using the 'undo planned move' option with the Gryphon Rider that it leaves bits of its frames behind - parts of the sprite that go beyond the edges of the hex.

That might be #3325.

**Comment by Wedge009** [comment]
Yes, you're right, thanks.

**Comment by Wedge009** [comment]
While we resolved 3325 I don't think I ever worked out how to replicate the original issue. I debug-created a Gryphon Rider, entered planning mode, planned a move, and executed the move. I don't see anything strange looking about the animation. Also, I don't see any idle animations in planning mode - units are greyed and static while a planned move is sitting on the game board.

**Comment by jostephd** [comment]
> how to replicate the original issue

wesnoth -t

:create Dread Bat

Enable planning mode

Plan a move (don't execute it)

Look at the hex the Bat occupies before the move for a few seconds, you'll see the silhouette twitch.

**Comment by Wedge009** [comment]
Test scenario: check
Create Dread Bat: check
Enable planning mode: check
Plan a move: check
Looking at the origin hex where the bat silhouette is... I don't see any movement at all after the move is planned. ): (I waited for something like a minute.)

**Comment by jostephd** [comment]
Here you go - gzipped because github wouldn't let me upload an mp4 otherwise

[dread-bat.mp4.gz](https://github.com/wesnoth/wesnoth/files/3753779/dread-bat.mp4.gz)

**Comment by Wedge009** [comment]
Fascinating. I don't see anything like that at all - my origin bat is completely static.

**Comment by soliton-** [comment]
Check Preferences -> Display if you have the animations disabled.

**Comment by Wedge009** [comment]
I have unit standing and idle animations both enabled.

**Comment by jostephd** [comment]
Here, `Animate water` is disabled but `Animate map` and unit standing and idle animation are enabled.

**Comment by Wedge009** [comment]
(Confirmed supersedes Works for me)

**Comment by Wedge009** [comment]
I'm finally able to replicate this... sort of. Following the steps described before on 2019-10-22, I still don't have any movement in the bat silhouette at it's starting location. However, moving the mouse around and hovering over other units on and off, generating different 'reach maps' and causing redraws, I was able to observe the silhouette 'twitch' as described. But it _only_ ever twitches when I move the mouse around and (presumably) cause redraws.

Tested against 1.15.17+dev (1.16 beta).
